### PR TITLE
Add onclick

### PR DESCRIPTION
**Pull request by vicente-palomero**

### CODE REVIEW

File: src/sui-link/index.jsx [hunk 1]
```diff
@@ -24,7 +25,12 @@ export default class Link extends React.Component{
     );
 
     return (
-        <a className={customclass} href={this.props.url} title={this.props.title} target={this.props.target}>
+        <a className={customclass}
```
Comment (FranDelaCasa) [inline src/sui-link/index.jsx:28]: IMHO It's better all properties inline, the html returned would be a little weird with a link splitted in 4 lines.

File: src/sui-link/index.jsx [hunk 2]
```diff
@@ -24,7 +25,12 @@ export default class Link extends React.Component{
...
+           href={this.props.url}
+           title={this.props.title}
+           target={this.props.target}
+           onClick={this.props.handleClick}
```
Comment (FranDelaCasa) [inline src/sui-link/index.jsx:32]: What about if handleClick is not defined? what's the html returned? will it has an empty onClick property?

File: src/sui-link/index.jsx (hunk 1 above)
Comment (carlosvillu) [inline src/sui-link/index.jsx:28]: dont worry the html will be inline.

Comment (FranDelaCasa) [inline src/sui-link/index.jsx:28]: Cool! :)

**Comment by delacruz-dev** [comment]
Why do we need this?

File: src/sui-link/index.jsx (hunk 2 above)
Comment (vicente-palomero) [inline src/sui-link/index.jsx:32]: Following the example:

Element without onClick:

``` html
<div id="link">
  <a class="sui-Link sui-Link-customclass" href="http://google.com" title="Element title" target="_parent" data-reactid=".0">
    <span class="fa sui-Link-icon fa-beer" data-reactid=".0.0"></span>
    <span data-reactid=".0.1">Alquiler de pisos en Madrid Capital</span>
  </a>
</div>
```

Element with onClick:

``` html
<div id="link-with-click">
  <a class="sui-Link" href="#" title="Element title" target="_parent" data-reactid=".1">
    <span class="fa sui-Link-icon fa-beer" data-reactid=".1.0"></span>
    <span data-reactid=".1.1">Previous click: never</span>
  </a>
</div>
```

And after a click: 

``` html
<div id="link-with-click">
  <a class="sui-Link" href="#" title="Element title" target="_parent" data-reactid=".1">
    <span class="fa sui-Link-icon fa-beer" data-reactid=".1.0"></span>
    <span data-reactid=".1.1">Previous click: 9/12/2015 9:24:13</span>
  </a>
</div>
```

**Comment by vicente-palomero** [comment]
@danderu Because we need to handle the action before the link is opened.

**Comment by delacruz-dev** [comment]
Ok! :+1:

**Comment by FranDelaCasa** [comment]
:+1:

**Pull request merged**